features/facebookImport/src/locales/de/common.json

```diff
@@ -1,6 +1,7 @@
 {
     "title": "Importer",
     "source.your.facebook.data": "Quelle: Ihre Facebook Daten",
+    "source.trustful": "Source: very trustful publication",
```

Comment:
Although I am not sure if this is the time for text corrections at all, maybe we can mark the missing translation with placeholders? (e.g.. "TO DO DE"). No clue if that makes sense